test/e2e_node/jenkins/benchmark/jenkins-benchmark.properties

```diff
@@ -0,0 +1,9 @@
+GCE_HOSTS=
+GCE_IMAGE_CONFIG_PATH=test/e2e_node/jenkins/benchmark/benchmark-config.yaml
+GCE_ZONE=us-central1-f
+GCE_PROJECT=test-kube-node-1348
```

Comment:
? This should be changed to https://github.com/kubernetes/kubernetes/blob/master/test/e2e_node/jenkins/jenkins-ci.properties#L4.